src/tiered/tiered_config.c

```diff
@@ -99,6 +99,7 @@ __wt_tiered_bucket_config(
     WT_ERR(__wt_config_gets(session, cfg, "tiered_storage.bucket", &bucket));
     if (bucket.len == 0)
         WT_ERR_MSG(session, EINVAL, "table tiered storage requires bucket to be set");
+    WT_ERR(__wt_config_gets(session, cfg, "tiered_storage.bucket_region", &bucket_region));
```

Comment:
Alphabetize its processing after `bucket_prefix`. 
Also I suggest a comment saying that it isn't required.